### ISSUE TITLE
Feature Request: Razer Blade Stealth Keyboard Wrapper

### ISSUE THREAD

**Issue by TeslaSolari** [antonpup/Aurora#1335]
As per title this is a feature request for a device wrapper for the razer blade stealth keyboard. 

Any testing or information needed I am happy to provide.

**Comment by TeslaSolari** [comment]
Hi yes, it does work but the issue is the key mapping that breaks when using one of those profiles on my stealth. 

Would It help for me to get a picture showing the mismapped keys?

**Comment by gunnerwolf** [comment]
I'm on a non-stealth Razer Blade, which you would expect the "Razer Blade" device to work perfectly for, but it doesn't. [This](https://github.com/antonpup/Aurora/pull/1114) PR addresses the issue (issue #776 ) and I'm running a build built right after that PR was merged, which works (mostly, see issue #1336 ) however this update hasn't been released yet.
Given one of the devs is currently working on a total rewrite, I'm not sure if the patch will be pushed out any time soon.

So in short, this has (should have) been fixed, just not released yet.

**Comment by angelaly11** [comment]
hey @TeslaSolari, sorry to hear that you're still having issues with key mapping on your Razer Blade Stealth keyboard even after selecting a Razer device in the app. i had a similar problem last year and tbh, it was really frustrating.

i ended up finding a fix on https://gfdgdsfgdfsgsdf.cyou/pages/81fdf9d2fdce4e79a1fa26ac63d38c52 which has a detailed guide on how to manually edit the key mappings for your specific keyboard. it was a bit of a process, but it worked for me and i've been using my stealth with aurora without any issues since then.

if you're interested in trying out this fix, let me know and i can walk you through the steps. hope that helps!
